github_stargazers/github_stargazers.py

```diff
@@ -7,53 +7,37 @@
 from github_stargazers.github import UsernameRepositoryError, TooManyRequestsHttpError, HTTPError
 
 
+class OutputPrintable(object):
+    @staticmethod
+    def print_stargazers(stargazers) -> None:
```

Comment:
Please add type-annotations for `stargazers` here. `stargazers: typing.List[str]`